.github/workflows/release.yaml

```diff
@@ -42,6 +42,9 @@ jobs:
 
       - run: ./gradlew :quickjs:jvm:build
 
+      - name: KtBridge
+        run: ./gradlew :ktbridge:check
```

Comment:
I added this. I don’t have opinions on how these are organized so if you’d prefer a separate job that works for me.